src/components/AwaitingPayment.vue

```diff
@@ -48,11 +50,16 @@ export default {
     },
     showQr(): boolean {
       if (this.isDesktopDevice) {
+        this.trackAnalytics(AnalyticsEvent.WaitingForPaymentWithQr)
         return true
       }
       if (this.forceShowQr) {
+        this.trackAnalytics(AnalyticsEvent.WaitingForPaymentWithQr)
         return true
       }
+
+      // there is no logic for the button as it is always visible
+      this.trackAnalytics(AnalyticsEvent.WaitingForPaymentWithButton)
```

Comment:
I think it would make sense to just track a single event type here: AnalyticsEvent.WaitingForPayment

rather have "isDesktopDevice" "forceShowQr" and "QrVisible" as parameters on that event

Reply:
maybe add a dedicated event for when the "Show QR button is clicked" that's a case that theoretically should not be necessary unless we screw things up, so I want to be able to figure out when users are clicking that

Reply:
Add the new event to the "show qr code" and also added the additional properties for the WaitingForPayment event